### PR TITLE
Added environment variable to control segment events

### PR DESCRIPTION
**Pull request by devidask27**

### CODE REVIEW

**Review by anupbarve — COMMENTED** [review]
Could you please add the purpose of this environment variable in README.md as well ?

**Comment by devidask27** [comment]
> Could you please add the purpose of this environment variable in README.md as well ?

Done

**Review by anupbarve — DISMISSED** [review]

File: pkg/pmk/segment.go [hunk 1]
```diff
@@ -28,7 +29,9 @@ type NoopSegment struct {
 
 func NewSegment(fqdn string, noTracking bool) Segment {
 	// mock out segment if the user wants no Tracking
-	if noTracking {
+	envCheck := os.Getenv("PF9CTL_SEGMENT_EVENTS_DISABLE")
+	segmentEventDisabled , _:= strconv.ParseBool(envCheck)
+	if noTracking || segmentEventDisabled {
```
Comment (nikhilbandi) [inline pkg/pmk/segment.go:34]: Just a curious question. In which scenario(s) is the bool noTracking set to True?

Comment (anupbarve) [reply pkg/pmk/segment.go:34]: It is a functionality to provide more granularity for different commands wrt posting segment events.
Not used at this moment.

Comment (devidask27) [reply pkg/pmk/segment.go:34]: For now it is always false

**Review by anupbarve — APPROVED** [review]
Please broadcast within our team on how to use this and also encourage them to use it if not working with segment events.

**Pull request merged**